### DISCUSSION TITLE
Add Requirement info

### DISCUSSION THREAD

**Discussion by ewwink** [uNetworking/uWebSockets.js#95] [Q&A] [answered]
This is not issue just suggestion, you can close it after reading.

The story:
I'm new with Node.js, previously I have installed Node v8.xx and because I read in the Readme specific equirement I just installed uWS with `#binary` version but it fail to run and I got error message

```
This version of µWS is not compatible with your Node.js build
```

Then I search related issue and found issue #26 but no mention about minimum `NODE_MODULE_VERSION`, it should be mininum module version `64` or Node.js 10.0.0. Then I upgrade  nodejs to 32-bit version because there is `win32` in the uWS filename.  but still I get different error

```
uws_win32_64.node is not a valid Win32 application
```

I search again for related issue but not found for solution, Then I look into Makefile and I see it use node `win-x64` or 64-bit library, finally I change the Node.js 10.15.3 (LTS) to 64-bit version and problem solved. 

conclusion:
To run this app we need Node.js 10.x.x (64) or 11.x.x (67) 64-bit version, and to make not confused maybe it will be good to rename `uws_win32_xx.node` to `uws_win64_xx.node` or create both version in build.

and thanks for this great project.

**Comment by ghost** [comment, marked as answer]
You are overthinking this. win32 does not mean "32-bit windows", it's a legacy name stuck in time. We have had 64-bit computers for what, 19 years now?
